### DISCUSSION TITLE
Riverpod 1.0 – thank you Remi

### DISCUSSION THREAD

**Discussion by scopendo** [rrousselGit/riverpod#1149] [General]
A short message to @rrousselGit to say thank you specifically for Riverpod 1.0 – having progressed from Provider to Riverpod 0.x to Riverpod 1.0, I find the package to be even more enjoyable to use and easier to understand.  Thank you – `ref` is now firmly entrenched!  👏

And of course, thank you also to any others who may have contributed.

**Comment by rrousselGit** [comment]
Thanks! Glad to hear that we're moving in a good direction 😊
